examples/tutorials/TodoApp/ext/LoginPage.js

```diff
@@ -2,10 +2,12 @@ import React from 'react'
 import { Link } from 'react-router-dom'
```

Comment:
Since `tutorials/TodoApp` is made to exactly reflect the result of the TodoApp tutorial, it would be best to not add WaspSourceHeader to it at all, as it pollutes the code (it is not present in the tutorial itself).

Reply:
Okay will change that.